docs/docs/cmd/planner/task/task-set.md

```diff
@@ -61,16 +61,26 @@ m365 planner task set [options]
 `--appliedCategories [appliedCategories]`
 : Comma-separated categories that should be added to the task
 
+`--priority [priority]`
+: Priority of the task: Urgent, Important, Medium, Low. Or an integer between 0 and 10 (check remarks section for more info). Default value is Medium.
```

Comment:
on `task set`, there is no default value. It would seem otherwise that it will change the value if you do not explicitely set it

Reply:
You are right. Should be no default value ☺️